pkg/client/dtclient/config_client.go

```diff
@@ -35,7 +35,36 @@ import (
 	"strings"
 )
 
-func (d *DynatraceClient) upsertDynatraceObject(ctx context.Context, theApi api.API, objectName string, payload []byte) (DynatraceEntity, error) {
+// APIData holds information about the api of the config to be read or written on the server
+type APIData struct {
+	ID                           string
+	SingleConfiguration          bool
+	UrlPath                      string
+	SubPathAPI                   bool
```

Comment:
it would be really nice if at this point we don't need this information anymore, however, for now there a re a few checks for subpathAPI in the client 🤷